guides/IMPORT_EXISTING_DOCUMENTS.md

```diff
@@ -0,0 +1,67 @@
+The `firestore-meilisearch` script is for use with the official Firebase Extension [**Meilisearch**](https://github.com/firebase/extensions/tree/main/firestore-meilisearch).
+
+### Overview
+
+The import script can read all existing documents in a Cloud Firestore collection or in sub-collections and index them into an index in Meilisearch.
+
+#### Important notes
+
+- You must run the import script over the entire collection **_after_** installing the Meilisearch extension; otherwise the writes to your database during the import might not be exported to the your `${param:MEILISEARCH_INDEX_NAME}` index.
+
+- The import script may take up to a while to complete. If your collection is large, you may want to consider importing in batches.
+
+- You cannot use wildcard notation in the collection path (i.e. /collection/{document}/sub_collection}). Instead, you can use a collectionGroup query. To use a collectionGroup query, provide the collection name value as ${COLLECTION_PATH}, and set ${COLLECTION_GROUP_QUERY} to true.
```

Comment:
in the config file?

Reply:
No, in the command line:
```
npx firestore-meilisearch 
  --project <project_id> \
  --source-collection-path <collection_name> \
  --index <index_uid> \
  --batch-size <100/default=300> \
  --host <host_address> \
  --api-key <api_key> \
  --non-interactive 
```
This is the guide for the script only